demo/src/main.tsx

```diff
@@ -25,21 +25,21 @@ const clientId = (function () {
 
 // use this for local development with local realtime
 //
-// const ablyClient = new Ably.Realtime({
-//   authUrl: `/api/ably-token-request?clientId=${clientId}`,
-//   port: 8081,
-//   environment: 'local',
-//   tls: false,
-//   clientId,
-// });
-
 const realtimeClient = new Ably.Realtime({
```

Comment:
Fine for now, but we should probably comment this out again and switch to the other client constructor. At some point I want to add a flag here so we can just set an env var and it'll run based off of that :D

Reply:
Oh yes, reverting this. Sorry I keep committing this change!

+1 for a flag or something, this is annoying. It was just by far the quickest way to handle it the first time it was put in.